scripts/setup-dependencies.sh

```diff
@@ -9,12 +9,10 @@ end="\033[0m"
 set -e
 
 # install in a custom prefix rather than /usr/local. by default, this
-# chooses "prefix" directory alongside "scripts" where configure.sh
-# resides.
+# chooses "prefix" directory alongside "scripts" directory.
```

Comment:
The prefix should be recreated here (`mkdir -p` won't error or cause harm on directories that already exist).

Anyone who's manually skipping the install-build-tools.sh should still be able to move forward.

Reply:
ACK, will update